src/engine/qcommon/common.cpp

```diff
@@ -52,25 +52,27 @@ Maryland 20850 USA.
 #include "sys/sys_events.h"
 #include <common/FileSystem.h>
 
+Cvar::Cvar<bool> com_developer = Cvar::Cvar<bool>( "developer", "", Cvar::TEMPORARY, false );
+Cvar::Cvar<int> com_speeds = Cvar::Cvar<int>( "com_speeds", "", Cvar::NONE, 0 );
```

Comment:
I think you can remove the ` = Cvar::Cvar<…>` part of these lines

Reply:
nitpick.